core/sql/common/NAType.cpp

```diff
@@ -670,6 +671,84 @@ short NAType::convertTypeToText(char * text,	   // OUTPUT
 				 displayCaseSpecific);
 }
 
+short NAType::getMyTypeAsHiveText(NAString * outputStr)  // output
```

Comment:
So, what we are really getting is the Hive type associated with a Trafodion data type?